lang/chinese.txt

```diff
@@ -447,8 +447,7 @@ GUI_STEAMCATSEL="选择 Steam 类别"
 BUT_SEL="选择"
 BUT_CREATE="创建"
 BUT_TAGS="标签"
-GUI_WARNNSG1="Steam 需要重新启动以使新添加的游戏可用"
-GUI_WARNNSG2="当为新游戏选择了标签后，必须在 Steam 客户端运行'XXX'命令并确认后才能激活它们"
+GUI_WARNNSG1="Ensure the Steam Client is closed before any new Non-Steam Games are added."
```

Comment:
Is this supposed to be Chinese or that happens later? same question for the other language txts

Reply:
Translations are community managed, when a new string is added it remains untranslated until a community member can translate them. This string already existed, but since it changed, I replaced the langfile strings :-)